src/Resque/Component/Job/Model/JobInterface.php

```diff
@@ -67,6 +67,8 @@ public function getArguments();
      */
     public function encode();
 
+    public function getOriginQueue();
```

Comment:
We also use it in the automatic restarting process. And I don't see why the origin queue shouldn't be a property of the job itself. Maybe I am missing something....